2021/09/deploying-dotnet-5-app-iis.md

```diff
@@ -0,0 +1,74 @@
+---
+author: "Juan Pablo Ventoso"
+date: 2021-09-16
+title: "Deploying a .NET 5 app on IIS"
+tags:
+- dotnet
+- csharp
+- iis
+---
+
+![Puzzle](/blog/2021/09/deploying-dotnet-5-app-iis/puzzle-cropped.jpg)
```

Comment:
Can you please include a link to the original photo and the license so we know we have the right to use it?

Reply:
The link is below, on the "Photo" label (https://flic.kr/p/4riBj8). I added license information next to the author, is that ok or should I add anything else?

Reply:
@juanpabloventoso Oh, duh, I just missed that!